dist/profile/manifests/kubernetes/resources/uplink.pp

```diff
@@ -0,0 +1,79 @@
+#   Class: profile::kubernetes::resources::uplink
+#
+#   This class deploys the Uplink app which is a simple  container
+#   container which communicates with a provisioned Azure PostgreSQL database
+#
+class profile::kubernetes::resources::uplink (
+    Array $clusters = $profile::kubernetes::params::clusters,
+    Array $domain_alias = [],
+    String $image_tag = 'latest',
+    String $domain_name = 'uplink.jenkins.io',
```

Comment:
I was expecting telemetry.jenkins.io. Is this deliberate?

Reply:
I would rather the domain name reflect the application that's running there. Since this isn't a general purpose telemetry service, I don't see any reason to call it as such.
:man_shrugging:

Reply:
Hence `jira.jenkins.io` and `confluence.jenkins.io` 😉